engine/runtime/src/main/java/org/enso/interpreter/service/ExecutionService.java

```diff
@@ -171,7 +171,7 @@ public void execute(
       interopLibrary.execute(call);
     } finally {
       context.getThreadManager().leave(p);
-      listener.dispose();
+        listener.dispose();
```

Comment:
Why this additional indent?

Reply:
Applied `javafmt` and produced: [f9cc871](https://github.com/enso-org/enso/pull/3509/commits/f9cc8711512c86f6f00f71b58734b6b6c4675e52)